src/helpers/discord_utils.ts

```diff
@@ -461,8 +461,10 @@ export function areUserAndBotInSameVoiceChannel(message: Eris.Message): boolean
  * @param message - The Message object
  * @returns the voice channel that the message's author is in
  */
-export function getVoiceChannelFromMessage(message: GuildTextableMessage): Eris.VoiceChannel {
-    const voiceChannel = (message.channel as Eris.TextChannel).guild.channels.get(message.member.voiceState.channelID) as Eris.VoiceChannel;
+export function getUserVoiceChannel(message: GuildTextableMessage): Eris.VoiceChannel {
+    const voiceChannelID = message.member.voiceState.channelID;
+    if (!voiceChannelID) return null;
+    const voiceChannel = state.client.getChannel(message.member.voiceState.channelID) as Eris.VoiceChannel;
```

Comment:
Nit: Why not `return (state.client.getChannel(message.member.voiceState.channelID) as Eris.VoiceChannel) || null;`   as a one liner for the function

Reply:
Passing `null` into `getChannel` throws an exception